panel/models/vtk/vtksynchronized.ts

```diff
@@ -0,0 +1,226 @@
+    this._vtk_renwin.getRenderer().resetCameraClippingRange()
+    this._vtk_renwin.getRenderWindow().render()
+    this.model.renderer_el = this._vtk_renwin
+    console.log('render end')
+  }
+
+  after_layout(): void {
+    console.log('after layout start')
+    super.after_layout()
+    console.log('after layout end')
+  }
+
+  _decode_arrays(): void {
+    console.log('decode arrays start')
+    const jszip = new (window as any).JSZip();
+    const promises: any = [];
+    const arrays: any = this.model.arrays;
+    const registerArray: any = this.registerArray;
+    const arrays_processed = this.model.arrays_processed;
+
+    function load(key: string) {
+        return jszip.loadAsync(atob(arrays[key]))
+            .then((zip: any) => zip.file('data/' + key))
+            .then((zipEntry: any) => zipEntry.async('arraybuffer'))
+            .then((arraybuffer: any) => registerArray(key, arraybuffer))
+            .then(() => arrays_processed.push(key));
+    }
+
+    Object.keys(arrays).forEach((key: string) => {
+        if (!this._decoded_arrays[key])
+        {
+            this._decoded_arrays[key] = true;
+            promises.push(load(key));
+        }
+    })
+
+    // Promise.all(promises).then(this._vtk_renwin.getRenderWindow().render)
+    console.log('decode arrays end')
+  }
+
+  _plot(): void{
+    console.log('plot start')
+    if(this._camera_callback){
+      this._camera_callback.unsubscribe()
+    }
+    this._synchronizer_context.setFetchArrayFunction(this.getArray)
+    this._vtk_renwin.getRenderWindow().setSynchronizedViewId(this.model.scene.id)
+    this._vtk_renwin.getRenderWindow().synchronize(this.model.scene)
+    this._vtk_renwin.getRenderWindow().render()
+
+    if(this._camera_callback){
+      this._camera_callback.unsubscribe()
+      this._camera_callback = null
+    }
+    this._camera_callback = this._vtk_renwin.getRenderer().getActiveCamera().onModified(
+      () => {
+          if(this._orientationWidget)
+            this._orientationWidget.updateMarkerOrientation()
+          this._vtk_renwin.getInteractor().render()
+      }
+    )
+    
+    console.log('plot end')
+  }
+
+  remove(): void {
+    console.log('remove start')
+    if(this._camera_callback){
+      this._camera_callback.unsubscribe()
+      this._camera_callback = null
+    }
+    super.remove()
+    console.log('remove end')
+  }
+
+  connect_signals(): void {
+    PanelHTMLBoxView.prototype.connect_signals.call(this) //super.super.connec_signals
+    this.connect(this.model.properties.orientation_widget.change, () => {
+      this._orientation_widget_visibility(this.model.orientation_widget)
+    })
+    this.connect(this.model.properties.arrays.change, () => this._decode_arrays())
+    this.connect(this.model.properties.scene.change, () => {
+      this._plot()
+    })
+    this.connect(this.model.properties.one_time_reset.change, () => {
+      this._vtk_renwin.getRenderWindow().clearOneTimeUpdaters()
```

Comment:
You can now call that [method](https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/Misc/SynchronizableRenderWindow/vtkObjectManager.js#L538)